docs/pages/fund-agents-apps/fund-your-app.mdx

```diff
@@ -66,6 +66,12 @@ Fund your payer wallet with USDC on Base. Here are some sources you can use to a
 
 To get USDC on Base Sepolia for use with XMTP testnet, you can use [https://faucet.circle.com/](https://faucet.circle.com/), which provides 10 USDC per hour.
 
+:::warning[Using testnet-staging?]
+
+If you are using the XMTP Funding Portal on testnet-staging, you must use mxUSD, not USDC. To get mxUSD, you must self-mint it. To learn how, see [For use with testnet-staging only: Self-mint mxUSD](#for-use-with-testnet-staging-only-self-mint-mxusd).
```

Comment:
do we want to explain what `mxUSD` stands for :)? It stands for `mock XMTP USD Fee Token`.